src/Components/AdvancedSearch/Internal/AdvancedSearchResultDetailsList/AdvancedSearchResultDetailsList.tsx

```diff
@@ -34,11 +35,16 @@ const AdvancedSearchResultDetailsList: React.FC<IAdvancedSearchResultDetailsList
 }) => {
     const { t } = useTranslation();
     const twinCount = twins.length;
-
     const classNames = getClassNames(styles, {
         theme: useTheme()
     });
-    const columns: IColumn[] = [
+    const [sortKey, setSortKey] = useState<keyof IADTTwin>('$dtId');
```

Comment:
can we move this '$dtId' to a const like `primarySortingKey` or something? it seems it is specialized key it would be nicer to have it in a separate value to easily change it in the future if we need to.

Reply:
I'd almost think this deserves to be a const at the app level like we do for PrimaryTwinName.